docs/java/features/resilience/caching.mdx

```diff
@@ -83,11 +83,11 @@ The above code will invoke `operation()` only once since the result of the first
 Only if the first call fails the second one would still happen.
 
 Note that a specific cache instance is not yet created when configuration is created.
-Instead it is only created when an operation is decorated or executed by the  `ResilienceDecorator`.
+Instead it is only created when an operation is decorated or executed by the `ResilienceDecorator`.
 
 :::tip
 The cache functionality is tenant aware by default.
-That means that by default cache entries created under a specific tenant will not be shared with other tenants.
+That means that by default, cache entries created under a specific tenant will not be shared with other tenants.
```

Comment:
The comma is not required and not common in english. If we want to improve readability we could write it in parentheses like this e.g. `That means that (by default), cache entries ...`

Reply:
Let's just remove the comma.